src/react/ErrorPage.jsx

```diff
@@ -28,12 +28,17 @@ class ErrorPage extends Component {
                 description="error message when an unexpected error occurs"
               />
             </p>
+            {message !== null ? (
```

Comment:
This check may not work - the prop will be `undefined` and not `null` if it hasn't been supplied.  If we add 
```
ErrorPage.defaultProps = {
  message: null,
};
```
At the end of the file, it'll then work.  Also, you may need to add `ErrorPage.propTypes` to pass the linter, too.

Reply:
Why not have this conditional be:

```jsx
{message && (
  <div role="alert">
    <p>{message}</p>
  </div>
)}
```

With this, the `div` would only be rendered if `message` is truthy (handling `null`, `undefined`, and empty string), and we wouldn't need to return the empty React fragment in the ternary conditional.